src/lib/y2network/interfaces.rb

```diff
@@ -16,36 +16,33 @@
 #
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
-require_relative "../test_helper"
-require "y2network/interface"
 
-describe Y2Network::Interface do
-  subject(:interface) do
-    described_class.new("eth0")
-  end
+require "forwardable"
 
-  describe "#==" do
-    context "given two interfaces with the same name" do
-      let(:other) { Y2Network::Interface.new(interface.name) }
+module Y2Network
+  # A container for network devices. In the end should carry methods for mass operations
+  # over network interfaces like old LanItems::find_dhcp_ifaces or so.
+  #
+  # FIXME: Intended for LanItems::Items separation
+  # proper cleanup is must
+  class Interfaces
```

Comment:
I would rather prefer a different name like `InterfacesList`, `InterfacesCollection`, `InterfacesSet`, or whatever you prefer.

We have something similar [in storage-ng](https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2storage/planned/devices_collection.rb). You could also extend it to support queries like this:

```ruby
interfaces.configured.type(:eth)
```
Note: actually, the yast2-storage counterpart should return a collection itself so we can chain methods.